main.go

```diff
@@ -83,11 +83,12 @@ func main() {
 				cli.IntFlag{
 					Name:  "timeout, t",
 					Value: 0,
-					Usage: "Set the timeout value when --wait flag is set",
+					Usage: "Set the timeout value when --wait flag is set (deprecated, use --wait <timeout> instead)",
 				},
-				cli.BoolFlag{
+				cli.IntFlag{
```

Comment:
This will result in a "hard deprecation", right?
Because any usage of the old variant (`BoolFlag`) will fail with a `parse error` for `--wait`, since no integer was passed in

Reply:
Yes, I agree with your observation. 

I am not sure how many existing '--wait' users out there, and how much problem it will cause them. It is pretty obvious when you type the commands by hand, but can be harder to debug when k3d is deeply embedded into other script or systems. Since k3d is such a young project I suspect there are not a lot of them out there -- I could be wrong. 

My hope is that we can quickly evolve the cli flags before 2.0 -- but try to maintain stability as much as possible post 2.0.  The effort you started on documentation will be very helpful in guiding the consistency and cli in general for 2.0 and beyond.

Having said that, I will be happy to implement what ever make sense here to ease the transition. Any suggestions?  We can always mention it in the release note, BTW.

Reply:
O.K. I saw your comment above. Will alias timeout to wait.

Reply:
As discussed offline, we merge as is.